src/endpoints/tokens/entities/token.ts

```diff
@@ -155,7 +155,11 @@ export class Token {
   @ApiProperty({ type: Number, nullable: true })
   totalVolume24h: number | undefined = undefined;
 
-  @Field(() => Boolean, { description: 'If the liquidity to market cap ratio is less than 1%, we consider it as low liquidity.', nullable: true })
+  @Field(() => Boolean, { description: 'If the liquidity to market cap ratio is less than 0.5%, we consider it as low liquidity.', nullable: true })
   @ApiProperty({ type: Boolean, nullable: true })
   isLowLiquidity: boolean | undefined = undefined;
+
+  @Field(() => Number, { description: 'If the liquidity to market cap ratio is less than 0.5%, we consider it as low liquidity and display threshold percent .', nullable: true })
```

Comment:
might remove references to '0.5%' since we can change it

Reply:
I think it would be good to leave the value in the description and if we change it in the future, we will change it in the ApiProperty as well

Reply:
ok 👍